android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -412,19 +412,13 @@ fun String.trimmed(): String {
     return this.take(MAX_URI_LENGTH)
 }
 
-/**
- * Returns a bitmap from the base64 representation of a PNG.
- * Returns null if the string is not a valid base64 representation of a PNG
- */
-fun String.base64PngToBitmap(): Bitmap? = base64ToBitmap(PNG_URI_PREFIX)
```

Comment:
nit: I think we should also remove `PNG_URI_PREFIX` as it looks it's not used anymore?

Reply:
Sorry I found it's still relevant!